pkgs/applications/file-managers/walk/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "walk";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "antonmedv";
+    repo = "walk";
+    rev = "v${version}";
+    sha256 = "sha256-lcXNGmDCXq73gAWFKHHsIb578b1EhznYaGC0myFQym8=";
+  };
+
+  vendorHash = "sha256-EYwfoTVcgV12xF/cv9O6QgXq9Gtc9qK9EmZNjXS4kC8=";
+
+  meta = with lib; {
+    description = "Terminal file manager";
+    homepage = "https://github.com/antonmedv/walk";
+    license = licenses.mit;
+    maintainers = with maintainers; [ portothree surfaceflinger ];
+  };
```

Comment:
Consider adding the meta.mainProgram attribute here. Especially when using the llama alias with lib.getExe that would make a difference.

Reply:
Do you think if this should be set explicitly to "walk" or reuse pname?

Reply:
Do not parameterize `pname`.